www/src/pages/docs/browser-apis.js

```diff
@@ -9,6 +9,10 @@ class BrowserAPIDocs extends React.Component {
     return (
       <Container>
         <h1 css={{ marginTop: 0 }}>Gatsby Browser APIs</h1>
+        <h2 css={{ marginBottom: rhythm(1 / 2) }}>Usage</h2>
+        <p>Implement any of these APIs by exporting them from a file named <code>gatsby-browser.js</code> in the root of your project.</p>
```

Comment:
@m-allanson probably want `marginBottom: rhythm(1)` or something along those lines as opposed to using `br` tags to control spacing.

Reply:
Good point, thanks 👍